README.md

```diff
@@ -1,24 +1,107 @@
 # TokenBridge Burner Wallet 2 Plugin Sample
 
 This sample plugin defines a mediator extension exchange pair to be used in the Exchange Plugin in the burner wallet. 
-It uses the resources from the `tokenbridge-plugin` package to define new assets and exchange pairs.
+It uses the resources from the `@poanet/tokenbridge-bw-exchange` npm package to define new assets and exchange pairs.
+
+### Introduction
+The [Burner Wallet 2.0](https://github.com/burner-wallet/burner-wallet-2) is a modular, extendable and customizable web application.
+It provides a set of packages and plugins that allow you to extend the basic functionality. Here we will focus on the Assets package and the Exchange Plugin.
+
+#### Assets
+You can add any native or ERC20 token to track its balance and send transactions. For example you can define a new ERC20 and add it to the wallet:
+```javascript
+import { xdai, dai, eth, ERC20Asset } from '@burner-wallet/assets';
+
+const test = new ERC20Asset({
+  id: 'tst', // The internal ID used by the wallet.
+  name: 'Test Token', // The display name that will be displayed to the user.
+  network: '1', // The chain ID of the chain the token is deployed to.
+  address: '0x52ad726d80dbb4A9D4430d03657467B99843406b' // Address where the token contract is deployed
+});
+
+const core = new BurnerCore({
+  assets: [test, xdai, dai, eth],
+});
+```
 
-### Clone the sample repo 
-Clone the sample repo to create a plugin project that uses the TokenBridge plugin. This sample repo containes a ready to publish plugin for a ERC677 to ERC677 bridge extension, but you can later modify it to define your own resources.
+The `ERC20Asset` class extends from the `Asset` class. The `Asset` class provides several methods to allow the wallet interact with the token.
+The following are the most important to pay attention and that could be overridden if your token behaves differently.
+- `getBalance`: Indicates how to get the balance of your asset.
+- `startWatchingAddress`: Indicates how to listen to value transfers incoming into your account.
+- `_send`: Indicates how to send value to another account.
+
+#### Exchange Plugin
```

Comment:
Extend the section with links to examples of the exchange plugins: xDai, Uniswap etc. Probably, it makes sense to add a short note for every example what is most interesting to pay attention (two-step interaction, transfer completion etc.)